src/domain/account.ts

```diff
@@ -114,6 +114,7 @@ function createMainAccount(
       accountID: MainAccountID, // main account is unique
       masterXPub: data.masterXPub,
       isReady: true, // always true for main account
+      template: `elwpkh($${MainAccountID})`,
```

Comment:
or then we need an additional field `descriptor` which uses the real thing we could import into Elements core for example?

```suggestion
      template: `elwpkh($${MainAccountID})`,
      descriptor: `elwpkh(${data.masterXPub})`
```